common/lib/xmodule/xmodule/tests/test_word_cloud.py

```diff
@@ -1,35 +1,46 @@
 # -*- coding: utf-8 -*-
 """Test for Word cloud Xmodule functional logic."""
 
+import json
 
+from django.test import TestCase
+from mock import Mock
 from webob.multidict import MultiDict
+from xblock.field_data import DictFieldData
 
-from xmodule.word_cloud_module import WordCloudDescriptor
+from xmodule.word_cloud_module import WordCloudBlock
+from . import get_test_system
 
-from . import LogicTest
 
+class WordCloudBlockTest(TestCase):
+    """Logic tests for Word Cloud XBlock."""
```

Comment:
Since the pull request is against upstream and a slight change has already been done to that, would it be possible to convert those docstrings to multiline docstrings? 

I don't want this request to cause merge conflicts, so if you consider it would cause trouble for people who are working on this repository before your pull request is merged, then forget about it. Otherwise, please let know so I can add a comments on the other single docstrings that were changed.

Reply:
According to https://www.python.org/dev/peps/pep-0008/#id33 single line docstrings are fine too.

Reply:
@symbolist The edx Python style guide extends PEP-8 and prescribes multi-line docstrings all the time:
- https://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/python-guidelines.html#docstrings